website/docs/security/authentication.md

```diff
@@ -6,46 +6,48 @@ description: Learn to use authentication mechanisms
 
 # Authentication
 
-FerretDB does not store authentication information (usernames and passwords) itself but uses the backend's authentication mechanisms.
-The default username and password can be specified in FerretDB's connection string,
-but the client could use a different user by providing a username and password in MongoDB URI.
-For example, if the server was started with `postgres://user1:pass1@postgres:5432/ferretdb`,
-anonymous clients will be authenticated as user1,
-but clients that use `mongodb://user2:pass2@ferretdb:27018/ferretdb?tls=true&authMechanism=PLAIN` MongoDB URI will be authenticated as user2.
-Since usernames and passwords are transferred in plain text,
+FerretDB supports `PLAIN`, `SCRAM-SHA-256` and `SCRAM-SHA-1` authentication mechanisms.
+A user with supported mechanism is created by `createUser()` command and stored in the `admin.system` database `users` collection.
+
+FerretDB uses passed username and password to authenticate against stored credentials.
+For example, if a client connects as `mongodb://user1:pass1@ferretdb:27018/ferretdb?tls=true&authMechanism=PLAIN`,
+`user1` is authenticated against its credential in `admin.system` database `users` collection.
+
+Before the first user is created, the credential passed in the connection string is used to connect directly to the postgreSQL backend.
+For example, when `admin.system` database `users` collection is empty and
```

Comment:
```suggestion
For instance, when the `admin.system.users` collection is empty and
```

Reply:
Is that the same meaning as `admin.system` database `users` collection? If that's the case should I replace with that everywhere?

Reply:
I'm assuming the users collection is inside the `admin.system` db. I left the first one, which sets the context for other usages. So you don't necessarily need to do that in all places

Reply:
I was wrong, it was `admin` database `system.users` collection. Updated that :see_no_evil: